controllers/front/OrderController.php

```diff
@@ -150,7 +151,12 @@ protected function bootstrap()
     protected function saveDataToPersist(CheckoutProcess $process)
     {
         $data = $process->getDataToPersist();
-        $data['checksum'] = $this->cartChecksum->generateChecksum($this->context->cart);
+        
+        try {
+            $data['checksum'] = $this->cartChecksum->generateChecksum($this->context->cart);
+        } catch (Exception $e) {
```

Comment:
You shouldn't suppress base exceptions. If you need to, you should throw a specific type of Exception (aka a subclass of Exception) so you catch the exception you're expecting, and not *any exception*.